tla/consensus/ccfraft.tla

```diff
@@ -1601,12 +1601,12 @@ StateTransitionsProp ==
 
 MembershipStateTransitionsProp ==
     [][\A i \in Servers :
-        \* RetirementCompleted is the terminal state
-        membershipState[i] = RetirementCompleted 
-        => membershipState[i]' = RetirementCompleted]_vars
+        membershipState[i] = RetiredCommitted
+        => membershipState[i]' = RetiredCommitted]_vars \* Terminal state
```

Comment:
Is this the same as `membershipState'[i]`?

Reply:
It doesn't matter if `i` is constant-level, but the semantics of both expressions are different if `i` is state-level.

Reply:
Ah so would `configuration[commitIdx[i]]'` be the same as `configuration'[commitIdx[i]']` and configuration'[commitIdx'[i]]` then?

Reply:
The idea of using TLAPS had not even crossed my mind. This is eye opening, thank you!

Reply:
I think we should still adopt a consistent `foo'[i]` style everywhere we can, but I will do that in a separate PR.